assignment1.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Assignment 1</title>
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<body>
+  <!-- creating a form -->
+  <h1>Form</h1>
+
+  <form action="assin.php" method="post">
+    <!-- using post method to send the data to assin.php file -->
+
+    First name : <input type="text" name="fname" id="fname"><br><br>
+
+    Last name : <input type="text" name="lname" id="lname"><br><br>
+
+    Full name : <input type="text" id="fullname" name="fullname" disabled><br><br>
```

Comment:
Disabled attribute is not the best choice, change this with appropriate attribute. Also reply the difference between the disabled attribute and which you need to use here.

Reply:
Disabled:
We can sent when the according form submits
elements do not receive focus.


Readonly:
We can't sent when the according form submits
elements can receive the focus.